packages/paste-core/layout/grid/src/Column.tsx

```diff
@@ -32,29 +35,37 @@ export const getColumnStyles = (props: ColumnProps): ColumnStyleProps => {
   return columnStyles;
 };
 
-const StyledColumn = styled.div(compose(space, flexbox, layout)) as React.FC<ColumnProps>;
+// @ts-ignore can't work out how to stop the styled div color prop from emotion clashing with our color style prop in BoxProps
+const StyledColumn = styled.div<StyledBoxProps>(
+  compose(space, flexbox, layout),
+  getCustomElementStyles
+) as StyledComponent<
+  Omit<React.DetailedHTMLProps<React.HTMLAttributes<HTMLElement>, HTMLElement>, 'color'>,
+  ColumnProps,
+  Record<string, unknown>
+>;
 
 const Column = React.forwardRef<HTMLDivElement, ColumnProps>(
-  ({as, children, count, gutter, offset, span, stretchColumnContent, vertical}, ref) => {
+  ({as, children, count, element = 'COLUMN', gutter, offset, span, stretchColumnContent, vertical, ...props}, ref) => {
     const ColumnStyles = React.useMemo(
       () => getColumnStyles({count, gutter, offset, span, stretchColumnContent, vertical}),
       [count, gutter, offset, span, stretchColumnContent, vertical]
     );
     return (
-      <StyledColumn {...ColumnStyles} as={as} ref={ref}>
+      <StyledColumn {...safelySpreadBoxProps(props)} {...ColumnStyles} as={as} data-paste-element={element} ref={ref}>
```

Comment:
Why is this `data-paste-element={element}` instead of `element={element}`?

Reply:
I originally did this because Column was not using Box. Its just a styled div. So I used `data-paste-element` directly since that what Box is doing. This bothered me some, so I just spent some time refactoring Column to use Box instead. That way I can just use `element` from Box. I'll push this change soon.

Reply:
Makes sense, nice!

Reply:
> This bothered me some, so I just spent some time refactoring Column to use Box instead.

Is also part of the AC to refactor anything not using Box to use Box